features/video-transformation/webhook.md

```diff
@@ -0,0 +1,197 @@
+| **data.transformation.options.audio_codec**            | `string?`   | `“aac”\|"opus"`                                                                            |
+| **data.transformation.options.format**                 | `string`    | `“mp4”\|”webm”\|”jpg”\|”png”\|”webp”`                                                      |
+| **data.transformation.options.stream_protocol**        | `string?`   | ABS protocol. `"HLS"\| "DASH"`                                                             |
+| **data.transformation.options.variants**               | `string[]?` | ABS protocol. `Array<“360“\|480”\|”720”\|”1028”>`                                          |
+| **data.transformation.options.auto_rotate**            | `boolean`   | Autorotate input images & videos based on their metadata, if no `rt` param applied on them |
+| **data.transformation.output.video_metadata.duration** | `number`    | Output duration in seconds (Float number)                                                  |
+| **data.transformation.output.video_metadata.bitrate**  | `number`    | Output bitrate (bits/second)                                                               |
+| **data.transformation.output.video_metadata.width**    | `number`    | Output width (no rotation meta data expected)                                              |
+| **data.transformation.output.video_metadata.height**   | `number`    | Output height (no rotation meta data expected)                                             |
+| **data.transformation.output.url**                     | `string`    | URL to download output file                                                                |
+| **data.timings.download_duration**                     | `string`    | Duration of download in milliseconds                                                       |
+| **data.timings.encoding_duration**                     | `string`    | Encoding of download in milliseconds                                                       |
+
+Example
+
+```js
+{
+  type: "video.transformation.ready",
+  id: "f30b0f2f-145e-4591-8e56-a6d0168fe123",
+  created_at: "2022-05-25T08:53:28.143Z",
+  request: {
+    x_request_id: "123456712",
+    url: "https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4",
+    user_agent: "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.122 Safari/537.36",
+  },
+  timings: {
+    donwload_duration: 3351,
+    encoding_duration: 2353,
+  },
+  data: {
+    asset: {
+      url: "https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4",
+    },
+    transformation: {
+      type: "video-transformation",
+      options: {
+        video_codec: "vp9",
+        audio_codec: "opus",
+        auto_rotate: true,
+        quality: 10,
+        format: "webm",
+      },
+      output: {
+        url: "https://www.learningcontainer.com/wp-content/uploads/2020/05/sample-mp4-file.mp4",
```

Comment:
Use this URL `https://ik.imagekit.io/demo/sample-video.mp4` in all examples.
Also, would it be better to showcase an example where transformations were being applied so that the user could see the difference between different properties? Right not `url`, `asset.url` all are same.

Reply:
done